### PR TITLE
Update card design

### LINKED ISSUE #25: Update grid item layout
### Kind of issue

- [ ] Improvement
- [X] Design
- [ ] Bug
- [ ] Other, namely:

### Description

This issue tracks various suggestions for redesign to grid items. Other suggestions can be provided as comment here or as a separate issue.

- As per [this comment](https://github.com/simple-icons/simple-icons/issues/980#issuecomment-752941407) (and more feedback): move or remove the logo in the background of grid items.
- As per [this comment](https://github.com/simple-icons/simple-icons/discussions/4865#discussioncomment-352284): make the color more prominent in grid items.
- As per [this comment](https://github.com/simple-icons/simple-icons/discussions/4865?sort=new#discussioncomment-361403): reduce the size of grid items as much as possible (even if https://github.com/simple-icons/simple-icons-website/issues/17 is added).

### PR DESCRIPTION
**Pull request by ericcornelissen**
Closes #25 

### Description

Inspired by [@PeterShaggyNoble design "mocks"](https://user-images.githubusercontent.com/3742559/117554029-5fbf1880-b055-11eb-9153-d1ae4f922dd9.png), this brings the redesigned website a bit closer to the old website.

### Preview

| Current | New |
| --- | --- |
| image | image |

### Discussion

The motivation for these changes are based on [feedback we received](https://github.com/simple-icons/simple-icons/discussions/4865) and, in particular, the following points:

- The "logo in the background" looked nice to some and ugly to others. Ultimately, it was there just for some visual flare and it can thus be argued that it was just "wasting space". Hence, we decided go back to the layout where the SVG is displayed prominently at the top of the card.
- The "brand guidelines" and "license" links are now displayed between the SVG and the title, this makes the titles align better.
- The card footer is now much simpler and smaller, just displaying the color and download options. For the record, while I would be okay switching/using icons for the download buttons I did not do that here to avoid discussion, what is currently in this PR most closely resembles the current live redesign.

Some suggestions to those reviewing this PR: in the interest of switching our domain to the new design I think we should largely avoid focussing on details such as spacing and size here - if you're concerned about anything like this I suggest to open an issue (after this PR has been merged) for that instead.

Also, after this is merged we can get https://github.com/simple-icons/simple-icons-website/pull/55 merged as it relies on the removal of the "logo in the background".

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]
I'm happy with that - thanks for the update @ericcornelissen!
Will wait for @PeterShaggyNoble's feedback before merging 👌

**Review by PeterShaggyNoble — APPROVED** [review]
Looks great, Eric, nice work 👍🏻

**Pull request merged**
